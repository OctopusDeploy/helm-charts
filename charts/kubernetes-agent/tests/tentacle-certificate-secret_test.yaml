suite: "tentacle configuration"
templates:
  - templates/tentacle-certificate-secret.yaml
tests:
- it: "is not created when certificate value is not supplied"
  set:
    agent:
      certificate: ""
  asserts:
  - hasDocuments:
      count: 0
- it: "should match snapshot"
  set:
    agent:
      certificate: "abcd"
  asserts:
    - matchSnapshot: {}