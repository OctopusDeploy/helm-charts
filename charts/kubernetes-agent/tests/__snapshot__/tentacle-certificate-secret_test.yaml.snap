should match snapshot:
  1: |
    apiVersion: v1
    data:
      certificate: YWJjZA==
    kind: Secret
    metadata:
      name: octopus-agent-tentacle-certificate
      namespace: NAMESPACE
    type: Opaque
